### PR TITLE
Thunar: update to 4.16.2.

### PR DESCRIPTION
**Pull request by uw2021**
#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

**Comment by AngryPhantom** [comment]
Sorry, guys. Could you please tell why Thunar got downgraded from 4.17 to 4.16 ?

**Comment by ericonr** [comment]
4.17 is the unstable release.

**Comment by uw2021** [comment]
> Sorry, guys. Could you please tell why Thunar got downgraded from 4.17 to 4.16 ?

It's my bad, I was unaware that 4.17 was the development release. If you would like the 4.17 release I can send you the .xbps file. Apologies.

**Comment by AngryPhantom** [comment]
@uw2021
No no, it's OK :)
I didn't know that it's a dev release either.
I have 4.17 installed and put it on hold since it fixes an important issue for me (in which I was also involved on their gitlab).
Cheers.

File: srcpkgs/Thunar/template [hunk 1]
```diff
@@ -1,8 +1,7 @@
 # Template file for 'Thunar'
 pkgname=Thunar
-reverts="4.17.0_1"
```
Comment (sgn) [inline srcpkgs/Thunar/template:3]: Please don't remove the `reverts` until `4.18.0`

Comment (uw2021) [reply srcpkgs/Thunar/template:3]: Re-added that line and force pushed the change.

**Pull request closed without merging**